roles/ara_api/vars/Ubuntu.yaml

```diff
@@ -33,6 +33,6 @@ ara_api_postgresql_packages:
 
 ara_api_mysql_packages:
   - mariadb-client
-  - libmariadbclient-dev
+  - libmariadb-dev
```

Comment:
libmariadb-dev is super old in 18.04, see my investigation here: https://github.com/ansible-community/ara/issues/325#issuecomment-896246447

libmariadbclient-dev is the right library to use and works in 20.04 as well.

With that in mind, maybe we should leave things as-is and revisit if it does break or if we have reports of issues.